README.md

```diff
@@ -345,6 +345,13 @@ TYK_PMP_PUMPS_MONGOAGG_TYPE=mongo-pump-aggregate
 TYK_PMP_PUMPS_MONGOAGG_META_USEMIXEDCOLLECTION=true
 ```
 
+###### Self Healing
+The default max size of a document in MongoDB is 16Mb. In Mongo Aggregate, when we try to update a document, and the size reaches the limit, an error is received.
```

Comment:
I think this should be 16MB, not 16Mb.
Is the error something specific to Mongo Aggregate Pump, or just a standard thing with MongoDB? (I'm guessing it's the latter)

I think this line could definitely be re-written to be clearer - and can propose different words if you can confirm the above.

For example, to something like this:

By default, the maximum size of a document in MongoDB is 16MB. If we try to update a document that has grown to this size, an error is received.

Reply:
@andyo-tyk you're right, the error is just a standard limit set by MongoDB.

That new text looks good to me.